### ISSUE TITLE
CVE-2020-8908 in Guava v.28.2-jre, should upgrade to v.30.1.1

### ISSUE THREAD

**Issue by ssainz** [apache/drill#2260]
**Describe the bug**
CVE-2020-8908 in Guava v.28.2-jre, should upgrade to v.30.1.1

**To Reproduce**
Please check vulnerability section in : 
(https://github.com/google/guava/issues/4011)[https://github.com/google/guava/issues/4011]

**Expected behavior**
Upgrading to v30.1.1 will mitigate this vulnerability.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: all
 - Browser all
 - Version all

**Smartphone (please complete the following information):**

**Additional context**

**Comment by luocooong** [comment]
@ssainz  Hello, It seems to be resolved. #2202

**Comment by cgivre** [comment]
Unless there is any objection, I'm going to close this as it should be resolved in Drill 1.19.

**Comment by ssainz** [comment]
Hello there @luocooong , @cgivre - defect not fixed. Please check this line:

https://github.com/vdiravka/drill/blob/master/pom.xml#L49 

shared guava still refers to guava 28.2 , thus, CVE-2020-8908 remains in Drill 1.19.

**Comment by luocooong** [comment]
@ssainz YES. Thanks for the reminder. @vdiravka has already started the process of updating Drill shaded Guava.

**Comment by pjfanning** [comment]
Can this be closed?

**Comment by cgivre** [comment]
> Can this be closed?

I think this is still valid, but not sure.  @vdiravka do you know the current status?

**Comment by ssainz** [comment]
Hi - the shared guava seems is still 28.2, then not fixed yet ~
https://github.com/apache/drill/blob/master/pom.xml#L52
```
<shaded.guava.version>28.2-jre</shaded.guava.version>
```

**Comment by pjfanning** [comment]
https://github.com/google/guava/issues/4011 affects a deprecated method and it only affects Drill if it Drill uses the deprecated method in the shaded org.apache.drill.shaded.guava.com.google.common.io.Files class. I see no evidence that Drill uses this method.
